packages/trpc/server/routers/viewer/teams/validateUniqueInvite.handler.ts

```diff
@@ -0,0 +1,35 @@
+import { prisma } from "@calcom/prisma";
+
+import type { TrpcSessionUser } from "../../../trpc";
+import type { TValidateUniqueInviteInputSchema } from "./validateUniqueInvite.schema";
+
+type ValidateUniqueInviteOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: TValidateUniqueInviteInputSchema;
+};
+
+const validateUniqueInviteHandler = async ({ ctx, input }: ValidateUniqueInviteOptions) => {
```

Comment:
To check for duplicate invites as we don't fetch all the members now we would have to make a DB request to check if member has been invited